pandas-stubs/core/frame.pyi

```diff
@@ -160,8 +160,8 @@ class _LocIndexerFrame(_LocIndexer):
         | Callable[[DataFrame], IndexType | MaskType | list[HashableT]]
         | list[HashableT]
         | tuple[
-            IndexType | MaskType | list[HashableT] | Hashable,
-            list[HashableT] | slice | Series[bool] | Callable,
+            Iterable[HashableT] | slice | Hashable,
```

Comment:
> Matching `str` is fine here, because the second component of the tuple ensures multiple columns are selected.
> 
> Both `df.loc["row", ["col1", "col2", "col3"]]` and `df.loc[["r", "o", "w"], ["col1", "col2", "col3"]]` return DataFrame which is the only thing this overload ensures.

No, you are incorrect.  The first example could create a `Series`:
```python
>>> import pandas as pd
>>> df = pd.DataFrame({"x":[1,2,3], "y":[4,5,6]}, index=["a", "b",
 "c"])
>>> df
   x  y
a  1  4
b  2  5
c  3  6
>>> df.loc["a", ["x", "y"]]
x    1
y    4
Name: a, dtype: int64
>>> type(df.loc["a", ["x", "y"]])
<class 'pandas.core.series.Series'>
```

Reply:
Oh, right.